q-transform/main.py

```diff
@@ -0,0 +1,48 @@
+    cur_time = datetime.datetime.now()
+
+    #construct the argument parse and parse the arguments
```

Comment:
Executable  / scripts should be placed into the 'bin' directory and given an bit more specific of a name, see examples in that directory.